### PR TITLE
Vaishnavi/login-screen

### LINKED ISSUE #57: Login Page/Sign out button
Links to #51

### PR DESCRIPTION
**Pull request by vee-16**
Closes #57

Added Login Screen:  

Screenshot 2023-06-01 at 6 31 51 PM

And Logout button with username displayed:

Screenshot 2023-06-01 at 6 31 59 PM

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -59,45 +61,50 @@ function App() {
...
+              {isSidebarOpen ? '‹' : '›'}
+            </Button>
+          </span>
+          {isSidebarOpen && (
+            <div
+              className={`fixed left-0 top-0 z-[5] h-full w-full bg-slate-800 opacity-30`}
+              onClick={() => dispatch(closeSidebar())}
+            />
+          )}
+        </div>
```
Comment (Graywing13) [inline client/src/App.js]: I like what you did below with `<LoginScreen />`! Is it possible to move the main app code into a separate file like that too?

Comment (vee-16) [reply client/src/App.js]: Yeah great suggestion! I'll add this

File: client/src/components/user/Logout.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  const dispatch = useDispatch()
+  const handleLogout = (e) => {
+    e.preventDefault()
+    dispatch(logout())
+  }
+
+  return (
+    <div>
+      <Button
+        className='fixed bottom-2 bg-red-500 bg-opacity-80 text-white hover:bg-red-300'
```
Comment (Graywing13) [inline client/src/components/user/Logout.js:15]: If we add `m-2 w-full`, does that make the button span the width of the sidebar?

Comment (Graywing13) [reply client/src/components/user/Logout.js:15]: Edit: it's ok

**Review by Graywing13 — APPROVED** [review]
lgtm

File: client/src/components/user/Logout.js (hunk 2 above)
Comment (vee-16) [reply client/src/components/user/Logout.js:15]: that made it span the width of the page for me, also not sure if you ran the client on your end but did were your login/logout buttons grey?

**Pull request merged**